src/scripts/main.js

```diff
@@ -134,6 +134,7 @@ const send = function(url, query, variables, next) {
 	}
 
 	xhr.setRequestHeader('Content-Type', 'application/json;charset=UTF-8')
+	xhr.withCredentials = true
```

Comment:
Could be a breaking change for those not allowing credentials. Need to test that. Looks fine otherwise!

Reply:
You are right. This will break all reports if `Access-Control-Allow-Credentials` is missing or not set to `true`.
I'll add it as an opt-in

Reply:
PR updated to include a new option: `ignoreOwnSite`.